### PR TITLE
PR for Refactoring "helper" functions into shared file - DR

### PR DESCRIPTION
**Pull request by rocio-desantiago**
## Summary

- _(Summarize the changes that have been made to the platform)_
  - Several functions moved to shared helper and issues utils into: `src/applications/appeals/shared/utils/helpers.js` and `src/applications/appeals/shared/utils/issues.js`
   

    - >  `hasDuplicates`, `getLegacyAppealsLength`, `getIssueNameAndDate`, `isEmptyObject`, `appStateSelector`, `getItemSchema`, `processIssues`, `returnPhoneObject`, and `readableList`

- _(If bug, how to reproduce)_
  - not a bug
- _(What is the solution, why is this the solution)_
  - these functions are the same for both 995 and 996; therefore don't have to be called twice and be imported from existing shared directory
- _(Which team do you work for, does your team own the maintenance of this component?)_
  - Benefits Management & Decision Reviews

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#65800

## Testing done

- _Describe what the old behavior was prior to the change_
  - no behavior changes

## Screenshots

  - N/A

## What areas of the site does it impact?

Appeals

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### CODE REVIEW

File: src/applications/appeals/996/content/areaOfDisagreement.jsx [hunk 1]
```diff
@@ -110,3 +109,7 @@ export const AreaOfDisagreementReviewField = ({ children }) => {
     </div>
   ) : null;
 };
+
+AreaOfDisagreementReviewField.propTypes = {
+  children: PropTypes.node.isRequired,
+};
```
Comment (anniebtran) [inline src/applications/appeals/996/content/areaOfDisagreement.jsx:115]: ooh interesting — was this a linter suggestion or did you just notice that this was missing/needed?

Comment (anniebtran) [reply src/applications/appeals/996/content/areaOfDisagreement.jsx:115]: hmmm okay 🤔  i wonder if it shouldn't have `isRequired` chained onto it bc on line 102, i'm seeing `children?.props` which kind of suggests that `children` might not be there. thoughts?

Comment (rocio-desantiago) [reply src/applications/appeals/996/content/areaOfDisagreement.jsx]: good catch! I didn't consider the `?` and how it could mean it may not always exist!

**Review by anniebtran — APPROVED** [review]
LGTM!

**Review by anniebtran — CHANGES_REQUESTED** [review]
yikes, vscode sabotaged you lol

**Review by anniebtran — APPROVED** [review]
nice 🙌

**Pull request merged**